### PR TITLE
2608 - cookiecutter for anonymizer.

### PR DESCRIPTION
**Pull request by shiranr**
Add template for the anonymizer.

### CODE REVIEW

File: presidio-analyzer/pylintrc [hunk 1]
```diff
@@ -10,7 +10,7 @@ reports=no
 disable=import-error,too-few-public-methods,missing-docstring,locally-disabled,fixme,cyclic-import,too-many-arguments,invalid-name,duplicate-code,bad-continuation
 
 [FORMAT]
-max-line-length=89
```
Comment (SharonHart) [inline presidio-analyzer/pylintrc:13]: This file is irrelevant now

File: azure-pipelines.yml [hunk 2]
```diff
@@ -1,62 +1,80 @@
...
+              pipenv sync --dev --sequential
+              pipenv run python -m spacy download en_core_web_lg
+
+        - task: Bash@3
+          displayName: 'Linting: Analyzer'
+          inputs:
+            targetType: 'inline'
+            workingDirectory: 'presidio-analyzer'
+            script: |
+              pipenv run pylint presidio_analyzer
```
Comment (SharonHart) [inline azure-pipelines.yml:51]: Remove pylint

**Review by SharonHart — APPROVED** [review]

**Review by SharonHart — APPROVED** [review]

File: presidio-analyzer/pylintrc (hunk 1 above)
Comment (shiranr) [reply presidio-analyzer/pylintrc:13]: Yes, but it's under your commit. I don't want to do too much changes which are related to pylint and analyzer.

Comment (SharonHart) [reply presidio-analyzer/pylintrc:13]: Still, I've removed this file in my PR. 
You making changes in it now made my PR dependent in yours.
It's fine, I'll rebase again, but try to keep the PR related to the change, also with the pipeline reformatting

**Review by omri374 — APPROVED** [review]

**Pull request merged**